modules/apps/account/account-admin-web/package.json

```diff
@@ -1,14 +1,16 @@
 {
 	"dependencies": {
 		"frontend-js-web": "*",
-		"metal-dom": "2.16.8",
-		"metal-events": "2.16.8"
+		"metal-path-parser": "1.0.4",
```

Comment:
Why adding these dependencies? The only one I see it's being used in this module is `metal-state`

Reply:
Same as above, conflicts must have confused me.